### PR TITLE
Use PBKDF2 to turn password into AES key

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

**Comment by madninja** [comment]
I think you forgot to add the pbkdf2 lib to the rebar.confg apps to include for escriptize

File: src/wallet_cli_main.erl [hunk 1]
```diff
@@ -173,22 +175,26 @@ handle_cmd(Specs, Args, OptFun, Fun) ->
             usage(Specs)
     end.
 
-encrypt_keys(Keys=#{public := PubKey}, Password) ->
+encrypt_keys(Keys=#{public := PubKey}, Password, Iterations) ->
```
Comment (madninja) [inline src/wallet_cli_main.erl:178]: Why not take iterations from Opts?

**Review by madninja — CHANGES_REQUESTED** [review]

File: src/wallet_cli_main.erl (hunk 1 above)
Comment (Vagabond) [reply src/wallet_cli_main.erl:178]: We do?

**Comment by Vagabond** [comment]
Added escriptize.

File: src/wallet_cli_main.erl (hunk 1 above)
Comment (madninja) [reply src/wallet_cli_main.erl:178]: Whoops, sorry, was looking in the wrong spot

**Review by madninja — APPROVED** [review]

**Pull request merged**